app/src/main/java/com/example/taskyy/domain/data_transfer_objects/RegisterUserDTO.kt

```diff
@@ -0,0 +1,7 @@
+package com.example.taskyy.domain.data_transfer_objects
```

Comment:
As this is a data model, this should also go in your data package. This is something domain should not know about :)